ui/components/ReconciledObjectsTable.tsx

```diff
@@ -33,51 +33,56 @@ function ReconciledObjectsTable({
     data: objs,
     error,
     isLoading,
-  } = useGetReconciledObjects(automationName, namespace, automationKind, kinds, clusterName);
+  } = useGetReconciledObjects(
+    automationName,
+    namespace,
+    automationKind,
+    kinds,
+    clusterName
+  );
 
   return (
     <RequestStateHandler loading={isLoading} error={error}>
-      <div className={className}>
-        <DataTable
-          fields={[
-            {
-              value: "name",
-              label: "Name",
-            },
-            {
-              label: "Type",
-              value: (u: UnstructuredObject) => u.groupVersionKind.kind,
-              sortType: SortType.string,
-              sortValue: (u: UnstructuredObject) => u.groupVersionKind.kind,
-            },
-            {
-              label: "Namespace",
-              value: "namespace",
-              sortType: SortType.string,
-              sortValue: ({ namespace }) => namespace,
-            },
-            {
-              label: "Status",
-              value: (u: UnstructuredObject) =>
-                u.conditions.length > 0 ? (
-                  <KubeStatusIndicator
-                    conditions={u.conditions}
-                    suspended={u.suspended}
-                  />
-                ) : null,
-              sortType: SortType.number,
-              sortValue: statusSortHelper,
-            },
-            {
-              label: "Message",
-              value: (u: UnstructuredObject) => _.first(u.conditions)?.message,
-              sortType: SortType.string,
-              sortValue: ({ conditions }) => computeMessage(conditions),
-            },
-          ]}
-          rows={objs}
-        />
-      </div>
+      <DataTable
```

Comment:
I assume it's not strictly related to the original issue (was not mentioned in the PR description).